From b06e388f63551eb6ea341849690fe3c498dda15d Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Sat, 27 Sep 2014 19:49:56 +0800
Subject: [PATCH] MLK-9637-2 usb: otg: test: add otg_srp_reqd flag in otg_fsm

This patch adds flag otg_srp_reqd in struct otg_fsm to indicate A device
as host wants B-UUT to do SRP once it ends the session.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 8e5f1b6b12ec77ec591909b09727049ffff37b9c)
---
 include/linux/usb/otg-fsm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/usb/otg-fsm.h b/include/linux/usb/otg-fsm.h
index 7a9967f..73b09c9 100644
--- a/include/linux/usb/otg-fsm.h
+++ b/include/linux/usb/otg-fsm.h
@@ -111,6 +111,7 @@ struct otg_fsm {
 	/* OTG test device */
 	int tst_maint;
 	int otg_vbus_off;
+	int otg_srp_reqd;
 
 	/* Informative variables */
 	int a_bus_drop_inf;
-- 
1.8.0

